CRM/MembershipExtras/Page/EditContributionRecurLineItems.php

```diff
@@ -263,8 +263,11 @@ private function showNextPeriodTab() {
    * @return string
    */
   private function calculateNextPeriodStartDate() {
-    $nextPeriodStartDate = new DateTime(CRM_Utils_Array::value('start_date', $this->contribRecur));
-    $intervalLength = CRM_Utils_Array::value('frequency_interval', $this->contribRecur) * CRM_Utils_Array::value('installments', $this->contribRecur);
+    $numberOfInstallments = 1;CRM_Utils_Array::value('installments', $this->contribRecur);
```

Comment:
ops, it is a mistake  which I just fixed, and squashed. But I wouldn't replace it with CRM_Utils_Array::value() since it will output 0 if the installments value is 0.